docs/source/guide/zne-3-options.myst

```diff
@@ -0,0 +1,371 @@
+
++++
+
+#### The special case of odd integer scale factors
+
++++
+
+Both local and global folding, if applied uniformly to all the gates of `circuit`, produce a `scaled_circuit` that has 3 times more gates than the input `circuit`.
+This corresponds to the `scale_factor=3` setting. For example:
+
+```{code-cell} ipython3
+import cirq
+from mitiq import zne
+
+# Get a circuit to fold
+qreg = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(cirq.ops.H.on(qreg[0]), cirq.ops.CNOT.on(qreg[0], qreg[1]))
+print("Original circuit:", circuit, sep="\n")
+
+# Apply local folding
+scaled_circuit = zne.scaling.fold_gates_at_random(circuit, scale_factor=3)
+print("Locally folded circuit:", scaled_circuit, sep="\n")
+
+# Apply global folding
+scaled_circuit = zne.scaling.fold_global(circuit, scale_factor=3)
+print("Globally folded circuit:", scaled_circuit, sep="\n")
+```
+
+The same trick can be generalized to any odd integer `scale_factor`.
+In this case, folding functions apply the mapping $G \longrightarrow G (G^\dagger G)^ {(\text{scale_factor} - 1)/2}$. For example:
+
+```{code-cell} ipython3
+num_gates = len(list(circuit.all_operations()))
+for scale_factor in [1, 3, 5, 7]:
+    scaled_circuit = zne.scaling.fold_global(circuit, scale_factor)
+    scaled_num_gates = len(list(scaled_circuit.all_operations()))
+    print(f"For scale_factor={scale_factor}, the number of gates was scaled by {scaled_num_gates / num_gates}")
+```
+
+**Note:** *When `scale_factor` is an odd integer, the number of gates is scaled exactly as dictated by the value of `scale_factor`.
+In this case, since all gates are folded the same number of times, the three local folding functions 
+{func}`.fold_gates_at_random()`, {func}`.fold_gates_from_left()` and {func}`.fold_gates_from_right()` have the same (deterministic) effect.*
+
++++
+
+#### The general case of real scale factors
+
++++
+
+More generally, the `scale_factor` can be set to any real number larger or equal than one. In this case,
+Mitiq applies additional folding to a selection of gates (for local folding) or to a final fraction of the circuit (for global folding),
+such that the total number of gates is *approximately* scaled by `scale_factor`. For example:
+
+```{code-cell} ipython3
+num_gates = len(list(circuit.all_operations()))
+for scale_factor in [1.2, 1.4, 1.6, 1.8, 2.0]:
+    scaled_circuit = zne.scaling.fold_gates_at_random(circuit, scale_factor)
+    scaled_num_gates = len(list(scaled_circuit.all_operations()))
+    print(f"For scale_factor={scale_factor}, the number of gates was scaled by {scaled_num_gates / num_gates}")
+```
+
+**Note:** *As printed above, if `scale_factor` is not an odd integer and if the input circuit is very short, there can be a large error in the actual scaling of the number of gates.
+For this reason, when dealing with very short circuits, we suggest to use odd integer scale factors.*
+
++++
+
+For longer circuits, real scale factors are better approximated.
```

Comment:
I think a few lines describing why this is so might be helpful. Or it could be better to link to the references where this matter is already discussed.

Reply:
Done here d3c6cee